rotkehlchen/tests/unit/test_balancer.py

```diff
@@ -0,0 +1,303 @@
+        timestamp=Timestamp(1614150923),
+        location=Location.BALANCER,
+        token0=EthereumToken('SNX'),
+        token1=EthereumToken('WETH'),
+        amount0_in=AssetAmount(FVal('186.643670114899034112')),
+        amount1_in=AssetAmount(ZERO),
+        amount0_out=AssetAmount(ZERO),
+        amount1_out=AssetAmount(FVal('2.283067259355947642')),
+    ),
+]
+TEST_SWAPS_1 = [
+    AMMSwap(
+        tx_hash='0x3c457da9b541ae39a7dc781ab04a03938b98b5649512aec2a2d32635c9bbf589',  # noqa: E501
+        log_index=18,
+        address=deserialize_ethereum_address('0x029f388aC4D5C8BfF490550ce0853221030E822b'),  # noqa: E501
+        from_address=deserialize_ethereum_address('0x0000000000007F150Bd6f54c40A34d7C3d5e9f56'),  # noqa: E501
+        to_address=deserialize_ethereum_address('0x70985E557aE0CD6dC88189a532e54FbC61927BAd'),  # noqa: E501
+        timestamp=Timestamp(1607008178),
+        location=Location.BALANCER,
+        token0=EthereumToken('WETH'),
+        token1=EthereumToken('AAVE'),
+        amount0_in=AssetAmount(FVal('1.594533794502600192')),
+        amount1_in=AssetAmount(ZERO),
+        amount0_out=AssetAmount(ZERO),
+        amount1_out=AssetAmount(FVal('11.260286362820602094')),
+    ),
+    AMMSwap(
+        tx_hash='0x3c457da9b541ae39a7dc781ab04a03938b98b5649512aec2a2d32635c9bbf589',  # noqa: E501
+        log_index=24,
+        address=deserialize_ethereum_address('0x029f388aC4D5C8BfF490550ce0853221030E822b'),  # noqa: E501
+        from_address=deserialize_ethereum_address('0x0000000000007F150Bd6f54c40A34d7C3d5e9f56'),  # noqa: E501
+        to_address=deserialize_ethereum_address('0x7c90a3cd7Ec80dd2F633ed562480AbbEEd3bE546'),  # noqa: E501
+        timestamp=Timestamp(1607008178),
+        location=Location.BALANCER,
+        token0=EthereumToken('AAVE'),
+        token1=EthereumToken('WETH'),
+        amount0_in=AssetAmount(FVal('11.260284842802604032')),
+        amount1_in=AssetAmount(ZERO),
+        amount0_out=AssetAmount(ZERO),
+        amount1_out=AssetAmount(FVal('1.6169340389857445212')),
+    ),
+    AMMSwap(
+        tx_hash='0x5e235216cb03e4eb234014f5ccf3efbfddd40c4576424e2a8204f1d12b96ed35',  # noqa: E501
+        log_index=143,
+        address=deserialize_ethereum_address('0x029f388aC4D5C8BfF490550ce0853221030E822b'),  # noqa: E501
+        from_address=deserialize_ethereum_address('0x0000000000007F150Bd6f54c40A34d7C3d5e9f56'),  # noqa: E501
+        to_address=deserialize_ethereum_address('0x8982E9bBf7AC6A49c434aD81D2fF8e16895318e5'),  # noqa: E501
+        timestamp=Timestamp(1607008218),
+        location=Location.BALANCER,
+        token0=EthereumToken('SYN'),
+        token1=EthereumToken('WETH'),
+        amount0_in=AssetAmount(FVal('980.08365587152306176')),
+        amount1_in=AssetAmount(ZERO),
+        amount0_out=AssetAmount(ZERO),
+        amount1_out=AssetAmount(FVal('1.352902561458047718')),
+    ),
+]
+TEST_SWAPS_1_EXPECTED_TRADES = [
```

Comment:
Are all these trade constants same for these and the api tests? If so put them in a common place